### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by mce35**
Hi, 2 small changes:

- Ability to display text for switches instead of raw value, used to display door status in my case:
Before:
image
After:
image

- Display power instead of usage for P1 smart meter, and support for General/kWh:
Before:
image
After:
image

### CODE REVIEW

**Comment by lokonli** [comment]
I'm trying to reproduce your example of the switch status.

I get:
image

How did you configure the block to show the icon and small text?
What is the domoticz device Type and SubType you used?

File: js/components/dial.js [hunk 1]
```diff
@@ -1494,29 +1496,33 @@ var DT_dial = (function () {
     } else {
       me.min = choose(me.block.min, -10);
```
Comment (lokonli) [inline js/components/dial.js:1497]: Can you set the default min value to 0 for a kWh device, and -2000 for a P1 meter?

File: js/components/dial.js [hunk 2]
```diff
@@ -1494,29 +1496,33 @@ var DT_dial = (function () {
...
       me.max = choose(me.block.max, 10);
```
Comment (lokonli) [inline js/components/dial.js:1498]: Can you set max to 10000 (now it's in Watt, previously kW)

**Review by lokonli — CHANGES_REQUESTED** [review]
Can you look at my remarks?

File: js/components/dial.js (hunk 1 above)
Comment (mce35) [reply js/components/dial.js]: Done

File: js/components/dial.js (hunk 2 above)
Comment (mce35) [reply js/components/dial.js]: Done

**Comment by mce35** [comment]
> I'm trying to reproduce your example of the switch status.
> 
> I get: image
> 
> How did you configure the block to show the icon and small text? What is the domoticz device Type and SubType you used?

I have 2 devices:
image

And I am using the following in CONFIG.js:
```php
blocks[15] = { width: 3, type: 'dial', title: 'Portail', values: [ { idx: 297 } ] }
```

**Comment by lokonli** [comment]
ah, I understand. You are combining two devices (15 and 297). I'll test.

**Pull request closed without merging**